### DISCUSSION TITLE
[resolved] Obsługa centrum wiadomości

### DISCUSSION THREAD

**Discussion by daniel8702** [allegro/allegro-api#4495] [Suggestions - resolved]
Możliwość pobierania wiadomości / spory

Było by świetnie gdyby w konfiguracji można było ustawić webhookURL na który przychodziły by powiadomienia o nowej wiadomości od klienta lub sporze. Ale nie jest to must have. 

Możliwość pobierania samych wiadomości była by wystarczająca :)

**Comment by hahyes** [comment]
W sumie śmieszne bo mają gotowe API do tego na froncie, które wygląda niemal identycznie jak zasoby od dyskusji, ale nie, nie ma obsługi wiadomości, bo nie.

**Reply by Lukasz-Zurek** [reply]
@daniel8702 @hahyes Planujemy wdrożenie obsługi centrum wiadomości w naszym API, jednakże na ten moment nie jestem w stanie zadeklarować konkretnego terminu.

**Comment by AureliuszBrussy** [comment]
@daniel8702 @hahyes planujemy udostępnić zasoby do obsługi Centrum Wiadomości jeszcze w czerwcu lub w lipcu.

**Reply by Tomo2903** [reply]
Wiadomo coś już?

**Reply by AureliuszBrussy** [reply]
Udostępnimy w tym miesiącu zasoby do obsługi Centum Wiadomości. Czekaj na newsa - niebawem podzielimy się szczegółami.

**Comment by AureliuszBrussy** [comment]
Wczoraj udostępniliśmy zasoby do obsługi Centrum Wiadomości - więcej informacji w [newsie](https://github.com/allegro/allegro-api/issues/4727).
